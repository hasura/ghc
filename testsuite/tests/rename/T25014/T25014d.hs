-- Should not compile trying to export a name with the wrong parent
module T25014b (A (foo)) where

data A

foo = 1
