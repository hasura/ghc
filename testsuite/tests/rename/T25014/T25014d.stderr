T25014d.hs:2:17: [GHC-88993]
     The type constructor ‘A’ is not the parent of the identifier ‘foo’.
      Identifiers can only be exported with their parent type constructor.
     In the export: A(foo)